### PR TITLE
Mpu6050 kalynenko

### PR DESCRIPTION
**Pull request by kalydmysfs**
Part 1:
To Implement bash script for reading temperature from mpu6050 and printing it in C and F;

Part 2:
To add /sys interface in sample module;
Added temp_f file in /sys/class/mpu6050 directory for getting temperature in Farenheit;

### CODE REVIEW

**Review by yekovalyov — CHANGES_REQUESTED** [review]
Task for kernel module requires resolution 0.001 F for temperature.

File: MPU6050/temperature.sh [hunk 1]
```diff
@@ -19,3 +19,13 @@ echo "Value of WHO_AM_I register is : $who_am_i"
 echo "Turning on MPU6050"
 i2cset -y 0 0x68 107 0
 
+while true
+do
+TempH=$(i2cget -y 0 0x68 65)
+TempL=$(i2cget -y 0 0x68 66)
+Temp_abs=$(( -(((TempH << 8)+ TempL) ^ ((1 << 0x10) - 1)) ))
```
Comment (yekovalyov) [inline MPU6050/temperature.sh]: Is this conversation correct for positive values?

**Review by yekovalyov — CHANGES_REQUESTED** [review]

File: MPU6050/temperature.sh (hunk 1 above)
Comment (kalydmysfs) [reply MPU6050/temperature.sh]: It is not correct variant, but I fixed script and drivers errors? please, can you check please @yekovalyov

**Review by yekovalyov — APPROVED** [review]

File: MPU6050/temperature.sh (hunk 1 above)
Comment (yekovalyov) [reply MPU6050/temperature.sh]: Yes. This conversation is correct.

**Pull request merged**